backend/src/main/java/com/google/sps/servlets/FindRecipesServlet.java

```diff
@@ -15,36 +15,53 @@
 package com.google.sps.servlets;
 
 import com.google.gson.Gson;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
 import com.google.sps.data.Recipe;
 import com.google.sps.scraping.BBCGoodFoodRecipeScraper;
 import java.io.IOException;
+import java.io.File;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
 import java.util.ArrayList;
 import java.util.List;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
 /* Servlet that:
- * in Post request, returns a list of recommended recipes based on the ingredients in the request
+ *  returns a set number of recipes for the given ingredients
  */
 @WebServlet("/api/find-recipes")
-public class FindRecipesServlet extends AuthenticationServlet {
-  @Override
-  protected void get(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    // no get request
-  }
+public class FindRecipesServlet extends HttpServlet {
```

Comment:
Could you please make FindRecipesServlet extend AuthenticationServlet? This is to ensure unauthorised users don't end any requests. https://github.com/googleinterns/step-2020-recipe-finder/blob/master/backend/src/main/java/com/google/sps/servlets/FindRecipesServlet.java